de.fu_berlin.inf.dpp/src/de/fu_berlin/inf/dpp/ui/wizards/pages/ConfigurationSettingsWizardPage.java

```diff
@@ -259,10 +260,12 @@ protected void setInitialValues() {
     this.skypeUsageButton.setSelection(!skypeUsername.isEmpty());
     this.skypeUsernameText.setText(skypeUsername);
 
-    this.statisticSubmissionButton.setSelection(
-        StatisticManagerConfiguration.isStatisticSubmissionAllowed());
+    if (FeedbackManager.isFeedbackFeatureRequired()) {
```

Comment:
If you look above in the file all GUI elements regarding the feature are still displayed.